.github/workflows/generation.yml

```diff
@@ -0,0 +1,29 @@
+name: generation
+on: [ "push", "pull_request"]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install coveralls pylint pylint-django
+          pip install -r warehouse/requirements.txt
+      - name: Analysing code pylint
+        run: |
+          pylint --load-plugins=pylint_django --django-settings-module=warehouse.settings warehouse/warehouse_app/
+      - name: Coveralls
+        run: |
+          coverage run --source=warehouse warehouse/manage.py test warehouse_app
```

Comment:
There are still some files that shouldn't be checked by coverage service. As I've mentioned earlier - you don't need to test coverage of django generated files like `admin.py`, `manage.py`, `urls.py` etc.